src/engine/source/cmds/src/cmdTest.cpp

```diff
@@ -0,0 +1,267 @@
+#include "cmds/cmdTest.hpp"
+
+#include <atomic>
+
+#include <re2/re2.h>
+
+#include <hlp/hlp.hpp>
+#include <kvdb/kvdbManager.hpp>
+#include <logging/logging.hpp>
+#include <rxbk/rxFactory.hpp>
+
+#include "base/utils/getExceptionStack.hpp"
+#include "builder.hpp"
+#include "catalog.hpp"
+#include "protocolHandler.hpp"
+#include "register.hpp"
+
+namespace
+{
+std::atomic<bool> gs_doRun = true;
+} // namespace
+
+namespace cmd
+{
+void test(const std::string& kvdbPath,
+          const std::string& fileStorage,
+          const std::string& environment,
+          int debugLevel,
+          bool traceAll,
+          const std::vector<std::string>& assetTrace,
+          int protocolQueue,
+          const std::string& protocolLocation)
+{
+    // Init logging
+    // TODO: add cmd to config logging level
+    logging::LoggingConfig logConfig;
+    logConfig.logLevel = logging::LogLevel::Debug;
```

Comment:
why can't we configure log level in test ?

Reply:
```
$ ./main test -f ../test/assets/ --environment demo-environment -k /tmp/ -d 2 -t 1
The following argument was not expected: 2
Run with --help for more information.
```

Reply:
Since it is testing I left logging as debug, though it may be useful to include log level argument.

Regarding debug option is a flag, to specify debug mode 2 you need to include the flag 2 times:
```$ ./main test -f ../test/assets/ --environment demo-environment -k /tmp/ -dd -t 1```

Added more explicit description:
```
  -d,--debug [0]              Enable debug mode [0-2]. Flag can appear multiple times. No flag[0]: No debug, d[1]: Asset history, dd[2]: 1 + Full tracing.
```